backend/amd64/emit.mlp

```diff
@@ -963,6 +991,14 @@ let emit_instr fallthrough i =
       | High -> T0
     in
     I.prefetch is_write locality (addressing addr QWORD i 0)
+  | Lop(Ibeginregion) ->
+      I.mov (domain_field Domainstate.Domain_local_sp) (res i 0)
+  | Lop(Iendregion) ->
+      I.mov (arg i 0) r11;
+      I.sub (domain_field Domainstate.Domain_local_sp) r11;
+      I.add r11 (domain_field Domainstate.Domain_local_total);
```

Comment:
Is `Domain_local_total` only for stats tracking you mentioned in the PR description? If so, r11 won't be needed after the stats are removed and then `Proc.max_register_pressure` and `Proc.destroyed_at_operation` would also need updating, right?

Reply:
You're right. Since this feature is going away I just deleted it, making the changes you mention to Proc and simplifying this bit of codegen. (There's an old build with the feature if we ever want to compute this stat)